src/main.c

```diff
@@ -473,9 +490,18 @@ int main(int argc, char *argv[])
         {
             if (1 <= verbose_level)
             {
-                printf("Reset MCU\n");
+                if (mode == 4) // operating in mode 5
```

Comment:
When used to determine whether reset is controlled by the code, or triggered by the user, change `if (mode == 4)` to `if (MODE_RESET_INPUT_TRUE == (mode & MODE_RESET_INPUT))`

This happens in multiple places.  I'll just say `MODE_RESET_INPUT` in future comments to mean the same thing.